lib/board.rb

```diff
@@ -0,0 +1,44 @@
+require 'colorize'
+
+class Board
+  attr_reader :symb, :counter, :status, :b, :symbols
+
+  def initialize
+    @b = Array.new(3) { |i| Array.new(3) { |j| ((i * 3 + j + 1)).to_s.blue } }
+    @counter = 9
+    @symb = ''
+    @symbols = %w[X O]
+  end
+
+  def turn(symb, loc)
+    if !loc.to_s.match?(/[1-9]/) || loc.to_s.length != 1 || @b[(loc - 1) / 3][(loc - 1) % 3].match?(/[XO]/)
+      @status = false
+    else
+      @status = true
+      @b[(loc - 1) / 3][(loc - 1) % 3] = symb
+      if winner(@b)
+        @counter = 0
+        @symb = symb
+      end
+      @counter -= 1
+    end
+  end
```

Comment:
This methods is violating the Single Responsability Principle.
Refer to the previous review, so it should be splitted into 3 methods.